src/chrome/skin/icons/freebsd/hicolor/22x22/apps/chatzilla.png

```diff
@@ -0,0 +1 @@
+../../../../img/chatzilla22.png
```

Comment:
No I meant the contrary: link the directory instead of each file. I think you linked each file (may be wrong).